### PR TITLE
Update blockly core docs to include: new test info, high level info on fork, playground dependency update details

### PR DESCRIPTION
**Pull request by bcjordan**

### CODE REVIEW

File: blockly-core/README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
-# Blockly Core
+# Blockly Core (Code Studio fork)
 
 This is a fork of [Blockly](https://code.google.com/p/blockly/), an open source visual programming environment.
 
+Major additions and changes in this fork:
```
Comment (bcjordan) [inline blockly-core/README.md:5]: @joshlory @Bjvanminnen @breville @islemaster @Hamms any other major features/changes worth mentioning in this fork description section?

**Comment by breville** [comment]
Nice, and very timely (@cpirich).  New trashcan was the only one I could think of.

File: blockly-core/README.md (hunk 1 above)
Comment (islemaster) [inline blockly-core/README.md:5]: I got nuthin'

Comment (Hamms) [inline blockly-core/README.md:5]: - support for a special UI for unattached blocks
- improved support for multiple blockspaces on a single page
- improvements to block arrangement on initialization

**Pull request merged**